src/components/InventoryDetail/ApplicationDetails.js

```diff
@@ -14,11 +14,14 @@ const ApplicationDetails = ({ onTabSelect, appList, ...props }) => {
     const history = useHistory();
     const dispatch = useDispatch();
     const searchParams = new URLSearchParams(search);
-    const items = useSelector(({ entityDetails }) => entityDetails?.activeApps);
+    const items = useSelector(({ entityDetails }) => entityDetails?.activeApps.filter(({ isVisible }) => isVisible !== false));
```

Comment:
I would add another guard in here
```suggestion
    const items = useSelector(({ entityDetails }) => (entityDetails?.activeApps || []).filter(({ isVisible }) => isVisible !== false));
```